test/integration_test.py

```diff
@@ -715,19 +715,12 @@ def _get_url_unchecked(self,
                            *,
                            stream: bool = False
                            ) -> urllib3.HTTPResponse:
-        if url.host in self.authenticating_hosts:
-            http, text = self._http, 'contextual'
-        else:
-            http, text = self._plain_http, 'plain'
-        url = str(url)
-        log.info('%s %s using %s client...', method, url, text)
```

Comment:
I'd like to see in the IT logs which client was used. Is that still possible?

Reply:
[I included the type of the client in every request log](https://github.com/DataBiosphere/azul/pull/5626/files#diff-dc87462f3dccdbae011c37acb0e0d2e39555dc776a732ec4cef2d4d09dc778c0R56), so the IT will show `HTTPClient.request(...` for the plain client and `TDRClient.request(...` when the authorization context is active.